internal/graphicsdriver/opengl/defaultshader.go

```diff
@@ -267,15 +269,10 @@ void main(void) {
   color.rgb /= color.a + (1.0 - sign(color.a));
   // Apply the color matrix or scale.
   color = (color_matrix_body * color) + color_matrix_translation;
-  color *= varying_color_scale;
```

Comment:
Does this really work when a color matrix is used?

Reply:
NVM, this should work...

Reply:
Yes, I think this is correct with a color matrix too.